ExampleCode/C++/install.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#*****************************************************************************
+#  iSENSE C++ Class Install Script
+#*****************************************************************************
+
+printf "iSENSE C++ Class Install Script\n"
+printf "This script pulls all required dependencies of the iSENSE C++ Class\n"
+printf "Included are:\n"
+printf  "- Pulls down iSENSE class / includes\n"
+printf  "- Pulls down iSENSE examples / example apps\n"
+printf  "- Pulls down picojson - 3rd party JSON library that the class depends on\n"
+printf  "- Installs curl, libcurl\n"
+printf "Please only run this script once. When this script is finished, look for\n"
+printf "a directory named \"isense\" in your home directory (~/isense)\n"
```

Comment:
Nothing should break - I check for the existence of the isense directory, so I could just tell the user at that point there's no need to continue since once you've git cloned the repos, you have the code and there's no point in downloading them a second time. 
I guess though, if I update the code, I could allow them to redownload the code by running the script a second time.